src/commands/live.rs

```diff
@@ -6,11 +7,42 @@ use spinner_macro::spinner_wrap;
 #[spinner_wrap]
 pub fn status() -> Result<String> {
     Ok(match time_entries::current_entry(&super::get_session())? {
-        Some(entry) => format!("Clocked in since {}!", local_time_format(entry.start_time)),
+        Some(entry) => {
+            let mut msg = format!("Clocked in since {}", local_time_format(entry.start_time));
+
+            let regular_hours_in_seconds = (entry.regular_hours * 3600.0) as u64;
+            let regular_hours_formatted = format_seconds_as_human_readable(regular_hours_in_seconds);
+
+            let total_duration_in_seconds = (entry.duration * 3600.0) as u64;
```

Comment:
I kind of cheat here and instead of iterating through `breaks` and accumulating the lengths, I get the total duration of the time entry - not sure if there are cases where this value is not sufficient.